docs/spec/slashing/tombstone.md

```diff
@@ -0,0 +1,74 @@
+# Staking Tombstone
+
+## Abstract
+
+In the current implementation of the `slashing` module, when the consensus engine informs the state machine of a validator's consensus fault, the validator is partially slashed, and put into a "jail period", a period of time in which they are not allowed to rejoin the validator set.  However, because of the nature of consensus faults and ABCI, there can be a delay between an infraction occuring, and evidence of the infraction reaching the state machine (this is one of the primary reasons for the existence of the unbonding period).
```

Comment:
What does the potential evidence delay have to do with ABCI or consensus faults? It's more a consequence of unpredictable gossip layer delays and the ability of validators to selectively reveal double-signatures (e.g. to infrequently-online light clients).

I think it's worth characterizing some of those scenarios in more detail (doesn't have to be in this spec though).